website/docs/r/windows_virtual_machine_scale_set.html.markdown

```diff
@@ -0,0 +1,405 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_windows_virtual_machine_scale_set"
+sidebar_current: "docs-azurerm-resource-windows-virtual-machine-scale-set"
+description: |-
+  Manages a Windows Virtual Machine Scale Set.
+---
+
+# azurerm_windows_virtual_machine_scale_set
+
+~> **NOTE:** **This resource is in Beta** and as such the Schema can change in Minor versions of the Provider.
+
+~> **NOTE**: All arguments including the administrator login and password will be stored in the raw state as plain-text. Read more about sensitive data in state.
```

Comment:
Was there supposed to be a link to read more about sensitive data here?

Reply:
fixed, also for the Linux resource